pkg/venafi/cloud/connector.go

```diff
@@ -673,16 +672,9 @@ func normalizeURL(url string) (normalizedURL string, err error) {
 		url = apiURL
 		//return "", fmt.Errorf("base URL cannot be empty")
 	}
-	modified := strings.ToLower(url)
-	reg := regexp.MustCompile("^http(|s)://")
-	if reg.FindStringIndex(modified) == nil {
-		modified = "https://" + modified
-	} else {
-		modified = reg.ReplaceAllString(modified, "https://")
-	}
-	if !strings.HasSuffix(modified, "/") {
-		modified = modified + "/"
-	}
+
+	modified := util.NormalizeUrl(url)
+
 	normalizedURL = modified
```

Comment:
This step can be removed, there is no further modification to the url after returning from `util.NormalizeUrl()` function

Reply:
It's done